packages/react-native-editor/babel.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = function ( api ) {
 				'../../node_modules/@babel/plugin-proposal-async-generator-functions'
 			),
 			'@babel/plugin-transform-runtime',
+			'@babel/plugin-transform-named-capturing-groups-regex',
```

Comment:
The new block makes use of regex named capture groups, which is only supported in the version of Hermes available in RN 0.71 (see https://github.com/facebook/hermes/issues/696 for more details). This resulted in the following error:

```
bundle/android/App.text.js:3647:961: error: Invalid regular expression: Quantifier has nothing to repeat
...atch(/^https?:\/\/(?<host>video(?:\.word|s\.files\.word)?press\.com)(?:\/v|\/embed)?\/(?<guid>[a-zA-Z\d]{8})/);ret...
```

The use of `@babel/plugin-transform-named-capturing-groups-regex` is a workaround to that error.